README.md

```diff
@@ -20,32 +20,43 @@ AgroService is a Python package that provides access to IPM Web Services (at lea
 
 ## Install
 
+
+### Install pip  
+Create a virtual environment and activate it:
+```bash
+python -m venv .venv
+source .venv/bin/activate
+```
+Install the package:
+```bash
+pip install . # -e for editable mode
+```
+
```

Comment:
I will not recommend to install openalea with venv unless we are able to install everything with venv.
We can limit us to mamba